go.mod

```diff
@@ -83,5 +83,3 @@ require (
 )
 
 go 1.23.0
-
-toolchain go1.24.1
```

Comment:
This is the culprit. I'm not sure what command I ran to add it here (unintentionally), but we don't need it.

Reply:
It was likely my go module upgrades?

Reply:
And yes, the toolchain version was upgraded in one of the last go module PRs.